cfme/scripting/quickstart/__init__.py

```diff
@@ -112,11 +100,9 @@ def install_requirements(venv_path, quiet=False):
 
     venv_call(
         venv_path,
-        'pip', 'install',
+        'pip3', 'install',
```

Comment:
I think only `pip` is fine as we are pointing in venv

Reply:
it is, I was trying to be explicit throughout quickstart.